tests/lib/dirs.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# Default applies for: Ubuntu, Debian
+SNAPMOUNTDIR=/snap
+LIBEXECDIR=/usr/lib
+CORELIBEXECDIR=/usr/lib
```

Comment:
`CORELIBEXECDIR` is a constant so I'd rather not have a variable for it (more obvious what is going on)

As for others, perhaps we could use `UNDER_SCORE` for naming them (just personal preference)